src/app/CommandSender.h

```diff
@@ -170,6 +170,8 @@ class CommandSender final : public Messaging::ExchangeDelegate
         return AddRequestDataInternal(aCommandPath, aData, aTimedInvokeTimeoutMs);
     }
 
+    CHIP_ERROR FinishCommand(const Optional<uint16_t> & aTimedInvokeTimeoutMs);
+
```

Comment:
Why is this being moved?

Reply:
Move the visibility from private to public